src/invoice_utils/engine/_engine.py

```diff
@@ -57,6 +44,7 @@ def _process_item(self, item_no: int, item_tax: Decimal, item: InvoicedItem):
                     "name": name,
                     "value": round(val + item_price, 6)
                 })
+        # TODO: This line overwrites the item_tax mandatory parameter received
```

Comment:
@koosie0507 Can you check this out? at line 122, we create a vat variable given as the item_tax parameter, that is not used since here at line 48 we use a different logic.

Reply:
This line allows overwriting the default VAT. Don't change it please. Also, please see my other comment about #TODO comments.